log-configs/cluster.yaml

```diff
@@ -6,13 +6,11 @@ rotation:
 
 loggerTree:
   severity: Debug+
-  files:
-    - node.log
 
   handlers:
     - { name: "JSON"
       , filepath: "node.json"
       , logsafety: SecretLogLevel
-      , severity: Info
+      , severity: Debug+
```

Comment:
do we need to specify the severity here, as it seems to be the same as the default?

Reply:
Good point - probably not, but I think it's useful to explicitly set the value.